### PR TITLE
[Filebeat] Changes to text fields in logstash module

### PR DESCRIPTION
**Pull request by ycombinator**
This PR is an offshoot of conversations and decisions made in https://github.com/elastic/beats/pull/10372 w.r.t `text` fields, but scoped to the `logstash` module.

### CODE REVIEW

File: filebeat/module/logstash/slowlog/_meta/fields.yml [hunk 1]
```diff
@@ -3,22 +3,24 @@
   description: >
     slowlog
   fields:
-    - name: message
```
Comment (ruflin) [inline filebeat/module/logstash/slowlog/_meta/fields.yml]: Was this not populated because there is no error in CI?

Comment (ycombinator) [reply filebeat/module/logstash/slowlog/_meta/fields.yml]: That's correct. It was never being populated. We had this conversation before 😄: https://github.com/elastic/beats/pull/10372/files#r251585088

Comment (ruflin) [reply filebeat/module/logstash/slowlog/_meta/fields.yml]: Perfect. Too many field discussions :-)

**Comment by ycombinator** [comment]
jenkins, test this

**Pull request merged**